lib/plugins/aws/invokeLocal/index.js

```diff
@@ -140,7 +140,13 @@ class AwsInvokeLocal {
   getConfiguredEnvVars() {
     const providerEnvVars = this.serverless.service.provider.environment || {};
     const functionEnvVars = this.options.functionObj.environment || {};
-    return _.merge(providerEnvVars, functionEnvVars);
+    const mergedVars = _.merge(providerEnvVars, functionEnvVars);
+    return Object.keys(mergedVars)
```

Comment:
While a bit more verbose, I think it's fine - we can refactor it to `entries/fromEntries` after we drop support for Node 10.x.